tests/run_tests/apps_run_test.py

```diff
@@ -21,54 +24,152 @@
 # import pytest
 # pytest.skip("eliminating conflicting test.", allow_module_level=True)
 
-project = "./FlinFlon.geoh5"
-
-geoh5 = Workspace(project)
+PROJECT = "./FlinFlon.geoh5"
```

Comment:
why the all caps?

Reply:
From PEP8, constants should technically be in caps no?